src/com/palmergames/bukkit/towny/object/jail/JailReason.java

```diff
@@ -1,23 +1,16 @@
 package com.palmergames.bukkit.towny.object.jail;
 
-import com.palmergames.bukkit.towny.TownySettings;
-
 public enum JailReason {
 
-	MAYOR("msg_jailed_mayor", 1), // hours is unused, always specified by the mayor when jailing. 
-	OUTLAW_DEATH("msg_jailed_outlaw", TownySettings.getJailedOutlawJailHours()),
-	PRISONER_OF_WAR("msg_jailed_war", 99);
+	MAYOR("msg_jailed_mayor"), // hours and bail deprecated, books handled in own class.

```

Comment:
Comments should be changed, nothing was deprecated and bail was never part of the master branch's enum.

Reply:
Erroneous use of term deprecated from JailReason.java removed and replaced with more useful comments in new commit